src/pages/MainQuizScreen.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from 'react';
+import MyButton from '../components/UI/button/MyButton';
+import ProgressBar from '../components/UI/progress_bar/ProgressBar';
+import QuestionBlock from '../components/UI/question_block/QuestionBlock';
+import MySelect from '../components/UI/select/MySelect';
+import Timer from '../components/UI/timer/Timer';
+
+const handleButtonClick = () => {
+  console.log('Button clicked!');
+};
+
+const quizQuestions = [
+  {
+    id: 1,
+    questionText: 'The Earth is flat.',
+    options: [
+      { text: 'True', isCorrect: false },
+      { text: 'False', isCorrect: true },
+    ],
+  },
+  {
+    id: 2,
+    questionText: 'Which planet is known as the Red Planet?',
+    options: [
+      { text: 'Earth', isCorrect: false },
+      { text: 'Mars', isCorrect: true },
+      { text: 'Jupiter', isCorrect: false },
+      { text: 'Saturn', isCorrect: false },
+    ],
+  },
+  {
+    id: 3,
+    questionText: 'Humans share 50% of their DNA with bananas.',
+    options: [
+      { text: 'True', isCorrect: true },
+      { text: 'False', isCorrect: false },
+    ],
```

Comment:
let's refactor it a bit
the main idea is to split questions and answers
an example
```ts
const questions = [
  { id: 1, content: 'The Earth is flat.' },
  { id: 2, content: 'Which planet is known as the Red Planet?' },
  //...
];

const answers = [
  { id: 1, questionId: 1, option: 'True', },
  { id: 2, questionId: 1, option: 'False', },
  { id: 3, questionId: 2, option: 'Earth', },
  { id: 4, questionId: 2, option: 'Mars', },
  //...
];

const correctAnswers = [
  { answerId: 1, questionId: 1 },
  { answerId: 4, questionId: 2 }
];

const getAnswersByQuestionId = (questionId: number) => answers
  .filter(answer => answer.questionId === questionId)

const isCorrectAnswer = (questionId: number, chosenAnswerId: number) => {
  return  Boolean(correctAnswers.find(answer => answer.questionId === questionId && answer.answerId === chosenAnswerId))
}
```
what do you think?